products/sle/main.pm

```diff
@@ -325,16 +325,16 @@ if (get_var('ENABLE_ALL_SCC_MODULES') && !get_var('SCC_ADDONS')) {
         # Container module is missing for aarch64. Not a bug. fate#323788
         $addons .= ',contm' unless (check_var('ARCH', 'aarch64'));
         set_var('SCC_ADDONS', $addons);
-        set_var('PATTERNS',   'default,asmm,pcm');
+        set_var('PATTERNS', 'default,asmm,pcm') if !check_var('PATTERNS', 'all');
```

Comment:
What's about not setting variable if it's already set? So we could override the value still. Does it make sense?